packages/material-ui/src/ButtonBase/Ripple.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import Transition from 'react-transition-group/Transition';
+import { Transition } from 'react-transition-group';
```

Comment:
Does this produce the same size as path imports? I implemented esm support with path imports too.
```js
import Transition from 'react-transition-group/Transition';
```

Reply:
Not sure but I'd rather avoid path imports as much as possible.

Reply:
Me too. ReplaceTransition and CSSTransition will appear in bundle though. They need to drop propTypes in production and drop defaultProps to achieve treeshaking.

Reply:
That doesn't make sense. It's definitely not the case for our webpack test bundles: https://86429-23083156-gh.circle-artifacts.com/0/tmp/material-ui/scripts/sizeSnapshot/build/stats.json